howso/client/base.py

```diff
@@ -4202,15 +4220,18 @@ def react_group(
                 serialized_cases.append(serialize_cases(group, features, feature_attributes))
 
         if case_indices is not None:
-            util.validate_case_indices(case_indices)
+            for group_indices in case_indices:
+                util.validate_case_indices(group_indices)
```

Comment:
This utility will present a confusing warning since its meant for a single case_indices list not multiple.